Add operator manual instructions to use a ZAP context for scans (DataBiosphere/azul-private#300) (#7603)
DataBiosphere/azul: OPERATOR.rst

a report, close the ZAP application, and then repeat the steps above to start
each additional scan with a fresh authentication token.

.. _`create a new session`: #zap-context-and-sessions

ZAP Context and Sessions
""""""""""""""""""""""""

With the ZAP application open, and prior to running any scan, start a new

> session and import ``azul-zap-scan.context`` from the azul-private repo.

Reviewer: No reason to not make this a link, too.

Failure to do so will pollute the scan results with known false positives and
findings from the previous scan. A new session is created each time you launch

> ZAP. Alternatively, to manually open a new session select *File* from the

Reviewer: ```suggestion
ZAP. Alternatively, to manually open a new session, select *File* from the
```

application menu bar, and then select *New Session*.

> To import the context file select *File* from the app menu bar, followed by

Reviewer: ```suggestion
To import the context file, select *File* from the application menu bar, followed by
```

> *Import Context…* and then proceed to find the `azul-zap-scan.context file`_

Reviewer: ```suggestion
*Import Context…* and then proceed to find the `azul-zap-scan.context`_ file
```

and click *Open*. Confirm the context is *In Scope* by double-clicking the newly
imported context and ensuring the checkmark is present in the *In Scope*

> checkbox. Additionally, a red dot is added to the icon next to entry labeled

Reviewer: ```suggestion
checkbox. After clicking <insert button label here>, a red dot will be shown in the icon next to entry labeled
```

> checkbox. After clicking *OK*, a red dot will be shown in the icon next to entry

Reviewer: ```suggestion
checkbox. After clicking *OK*, a red dot will be shown in the icon next to the entry
```

> *Azul Context* when enabled. Delete the ``Default Context`` by right-clicking

Reviewer: ```suggestion
*Azul Context* when enabled. Lastly, delete the ``Default Context`` by right-clicking
```

> labeled *Azul Context* when enabled. Lastly, delete the ``Default Context`` by

Reviewer: ```suggestion
labeled *Azul Context*. Lastly, delete the ``Default Context`` by
```

> right-clicking the context and selecting the *Delete* option, thus disabling it.

Reviewer: ```suggestion
With the ZAP application open, and prior to running any scan, start a new session and import
``azul-zap-scan.context`` from the azul-private repo.
Failure to do so will pollute the scan results with known false positives
and findings from a previous scan. A new session is created each time you
launch ZAP. Alternatively, to manually open a new session select *File* from the application menu bar, and then select *New Session*.

To import the context file select *File* from the app menu bar, followed by
*Import Context…* and then proceed to find the ``azul-zap-scan.context`` file

REVIEW: Add link to the file

and click *Open*. Confirm the context is *In Scope* by double-clicking the newly
imported context and ensuring that the *In Scope* checkbox is set.

REVIEW: I don't understand the next sentence. I don't know what a subicon is.

Additionally, a red target subicon is present in the window icon
preceeding the context name when in scope. Delete the ``Default Context`` by
right-clicking the context and selecting the *Delete* option, thus disabling it.
```

Author: A red dot is added to the icon next to entry labeled *Azul Context* when enabled.

> right-clicking the context and selecting the *Delete* option, thus disabling it.

Reviewer: ```suggestion
right-clicking it and selecting the *Delete* option.
```

a report, close the ZAP application, and then repeat the steps above to start
each additional scan with a fresh authentication token.

.. _`create a new session`: #zap-context-and-sessions

ZAP Context and Sessions
""""""""""""""""""""""""

With the ZAP application open, you must start a new session prior to running a

> new scan and import ``azul-zap-scan.context`` from the azul-private repo.

Reviewer 2: ```suggestion
With the ZAP application open, you must start a new session and import
``azul-zap-scan.context`` from the azul-private repo prior to running a new scan .
```

Failure to do so can pollute the scan results with known false positive alerts
and with findings from the previous scan. A new session is created each time you
launch ZAP, or alternatively, to manually open a new session, from the app menu
bar select *File*, and then *New Session*.

To import the context file, from the app menu bar select *File*, followed by
*Import Context…* and then proceed to find the ``azul-zap-scan.context`` file
and click *Open*. Confirm the context is *In Scope* by double-clicking the newly
imported context and ensuring the checkmark is present in the *In Scope*
checkbox. Additionally, a red target icon is precent in the window icon

> preceeding the context name when is in scope. Be sure to delete the

Reviewer 2: ```suggestion
checkbox. Additionally, a red target icon is shown preceding the
context name in the main window when that context is in scope.
Be sure to delete the
```
